Formula/helix.rb

```diff
@@ -0,0 +1,22 @@
+class Helix < Formula
+  desc "Post-modern modal text editor"
+  homepage "https://helix-editor.com"
+  url "https://github.com/helix-editor/helix.git",
+       tag:      "v0.6.0",
+       revision: "ac1b7d8d0a608f47edfee2872d414e94fd26cc31"
+  license "MPL-2.0"
+  head "https://github.com/helix-editor/helix.git", branch: "master"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "helix-term")
+    prefix.install "runtime"
+    mv bin/"hx", bin/"hx-bin"
+    (bin/"hx").write_env_script(bin/"hx-bin", HELIX_RUNTIME: prefix/"runtime")
```

Comment:
```suggestion
    bin.env_script_all_files libexec/"bin", HELIX_RUNTIME: prefix/"runtime"
```
Is there any reason for you to be able to run `hx` without `HELIX_RUNTIME` set?

Reply:
Thanks for the suggestion. I was not aware of the convenient function provided by Homebrew lol.

As for the reason for setting `HELIX_RUNTIME `, it is mentioned in https://github.com/helix-editor/helix#installation that
> Helix also needs its runtime files so make sure to copy/symlink the `runtime/` directory into the
config directory (for example `~/.config/helix/runtime` on Linux/macOS, or `%AppData%/helix/runtime` on Windows).
This location can be overridden via the `HELIX_RUNTIME` environment variable.

We have to install the `runtime` directory to the cellar and set the environment variable since we can not install it to the users' home directory.

BTW, I moved the `runtime` directory to `libexec`.

Reply:
Judging by this it's more appropriate in `etc`

Reply:
> We have to install the `runtime` directory to the cellar and set the environment variable since we can not install it to the users' home directory.

I'm aware, but initially you installed `hx` without this environment variable set, which is why I asked if you ever want to be able to run `hx` without this set.

> Judging by this it's more appropriate in `etc`

Probably not, because we can't ship updates to files in `etc`